src/EnergyPlus/HVACManager.cc

```diff
@@ -2725,6 +2732,182 @@ namespace HVACManager {
 
 	}
 
+	void
+	CheckMassBalances()
+	{
+		// Using/Aliasing
+		using namespace Fans;
+		using namespace Pumps;
+		using DataHVACGlobals::SmallMassFlow;  // Air flow tolerance
+		using DataBranchAirLoopPlant::MassFlowTolerance;  // Liquid flow tolerance
+		using DataHVACGlobals::TimeStepSys; // system time step [hr]
+		using DataHVACGlobals::SysTimeElapsed; // elapsed time in time step [hr]
+		using DataGlobals::CurrentTime; // current simulation time [hr]
+		using General::CreateSysTimeIntervalString; // string representation of simulation time
+		using General::RoundSigDigits; // round to number of decimals
+
+		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
+		static Real64 CurrentEndTime( 0.0 ); // end time of time step for current simulation time step
+
```

Comment:
We don't want new static variables buried inside routines.  This needs to be moved up so it can be reinitialized in clear_state()